### PR TITLE
Add MicroProfile Rest Client Generator extension

### PR DESCRIPTION
**Pull request by kathrynkodama**
Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ By installing the MicroProfile Extension Pack, the following extensions are inst
 
 ### Creating a MicroProfile project
 * [MicroProfile Starter](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-starter-vscode-ext) - Quickly generate a MicroProfile Maven project.
+* [MicroProfile Rest Client Generator](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-rest-client-generator-vscode-ext) - Quickly generate a MicroProfile Rest Client interface template within an existing project.
```
Comment (rzgry) [inline README.md]: Should we mention that it uses an OpenAPI yaml file?

Comment (kathrynkodama) [reply README.md]: How about "Quickly generate a MicroProfile Rest Client interface template from an OpenAPI yaml file or url."

Comment (rzgry) [reply README.md]: I think thats better.

Comment (ericglau) [reply README.md]: Perhaps "from an OpenAPI document" or "from an OpenAPI specification"?  https://github.com/OAI/OpenAPI-Specification uses the term OpenAPI document, while https://swagger.io/docs/specification/about/ also describes it as an OpenAPI file or an OpenAPI specification.

**Pull request merged**